README.md

```
@@ -24,189 +18,463 @@ go build -o amp ./cmd/amp
-|------|-------|-------------|
-| `--force` | `-f` | Skip confirmation prompts |
+| `--verbose` | `-v` | Enable debug output |
+| `--help` | `-h` | Show help |
 
 ## Interactive Mode
 
-Running `amp` without arguments launches an interactive shell:
+Run `amp` without arguments to start an interactive shell:
 
-```bash
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)